docs/reference/channel.md

```diff
@@ -72,15 +72,15 @@ The `channel.fromLineage` factory creates a channel that emits files from the {r
 
 ```nextflow
 channel
-    .fromLineage(workflowRun: 'lid://0d1d1622ced3e4edc690bec768919b45', labels: ['alpha', 'beta'])
+    .fromLineage(workflowRun: 'lid://0d1d1622ced3e4edc690bec768919b45', label: ['alpha', 'beta'])
     .view()
 ```
 
 The above snippet emits files published by the given workflow run that are labeled as `alpha` and `beta`.
 
 Available options:
 
-`labels`
+`label`
 : List of labels associated with the desired files.
```

Comment:
Shouldn't this be `labels` if it's a list? I think `label` only makes sense on the command line since you have to specify it once for each label